permutations.r

```diff
@@ -1,22 +1,49 @@
-Packages <- c("gtools", "readr", "tibble", "dplyr")
 
-lapply(Packages, library, character.only = TRUE)
+################################################################################
+
+# WHAT IS THIS SCRIPT FOR?
+
+# OH GOD WHAT IS ANYTHING FOR?
+
+################################################################################
+
+packages <- c("gtools", "readr", "tibble", "dplyr")
+
+lapply(packages, library, character.only = TRUE)
```

Comment:
This is a fucking power move right here. I actually showed it off to some of my friends.

I have just a tiny thing to fuss about. It's generally good practice to keep things lowercase unless capital letters are really needed for clarity -- so I suggest changing `Packages `to `packages`.